ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,281 @@
+    - Lord external AHRS renamed to MicroStrain5
+    - MAV_CMD_EXTERNAL_POSITION_ESTIMATE supports setting approximate position during dead-reckoning
+    - Microstrain7 external AHRS support
+4) Driver enhancements
+    - 3DR Solo supports up to 6S batteries
+    - Airspeed health checks vs GPS use 3D velocity
+    - BDShot support on IOMCU (e.g. main outputs of boards with IOMCU)
+    - DroneCAN battery monitors calculate consumed energy if battery doesn't provide directly
+    - DroneCAN RC and Ghost RC protocol support
+    - EFI MAVLink driver
+    - Extended DShot Telemetry support (requires BLHeli32 ver 32.93, set SERVO_DSHOT_ESC=3)
+    - GPS L5 band health override to enable L5 signal use (see GPS_DRV_OPTIONS)
+    - GPS-for-yaw works at lower update rate (3hz minimum)
+    - GSOF GPS supports GPS_COM_PORT parameter
+    - Hirth ICEngine support
+    - ICE option to enable/disable starting while disarmed
+    - ICE support for starter via relay
+    - IMUDATA serial protocol outputs raw IMU data on serial port (only available using custom build server)
+    - Innomaker LD06 360deg lidar support
+    - Intelligent Energy fuel cells new protocol support
+    - IRC Tramp supports 1G3 bands A and B
+    - JAE JRE-30 radar
+    - KDECAN driver rewrite (params moved to KDE_, works on all vehicles)
+    - MCP9601 temperature sensor support
+    - NanoRadar NRA24 rangefinder support
+    - NeoPixelsRGB support
+    - NoopLoop TOFSense, TOFSenseF-I2c rangefinder support
+    - OSD shows flashing horizon when inverted
+    - OSD2 support (e.g. second OSD)
+    - QMC5883P compass support
+    - Relay refactored to support RELAYx_FUNCTION, RELAY_STATUS message support added
+    - Reventech fuel level support (extends existing analog driver, see BATT_FL_xx parameters)
+    - RPLidarS1 360 deg lidar support and improved reliability for all RPLidars
+    - SBF GPS supports yaw from dual antennas
+    - Temperature sensor using analog voltages supported
+    - Winch driver enhancements including stuck protection, option for spin-free on startup
+5) Control and navigation changes and enhancements
+    - Auto missions can always be cleared while disarmed (would fail if mission still running)
+    - DO_ENGINE_CONTROL allows starting engine even when disarmed
+    - DO_SET_MISSION_CURRENT command can reset mission (see Reset Mission field)
+    - DO_SET_SERVO, DO_REPEAT_SERVO work with servo outputs set to RCInxScaled
+    - Fractional Loiter Turn Support in missions
+    - HarmonicNotch supports up to 16 harmonics
+    - JUMP command sends improved text msg to pilot (says where will jump to)
+    - MAV_CMD_AIRFRAME_CONFIGURATION can control landing gear on all vehicles
+    - MOT_OPTIONS allows voltage compensation to use raw battery voltages (instead of current corrected voltage)
+    - PID controllers get DFF/D_FF (derivative feed-forward), NTF (target notch filter index) and NEF (error notch filter index)
```

Comment:
Might want to mention the new Filter library for setting these parameters

Reply:
ok thanks.  Could you give me a suggested wording?  Sorry to be lame but I don't know about the new filter library I'm afraid.

Reply:
PID notch filter configured via new filter library using FILT parameters

Reply:
ok, I've added this new line below the other two.  The existing lines are really long so not sure I can fit any more on them.